frontend/src/__tests__/cypress/cypress/utils/oc_commands/project.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Create an Openshift Project
+ *
+ * @param projectName Project Name
+ * @param displayName Project Display Name
+ * @returns Result Object of the operation
+ */
+export const createOpenShiftProject = (projectName: string, displayName?: string) => {
+  const finalDisplayName = displayName || projectName;
+  const ocCommand = `oc new-project ${projectName} --display-name='${finalDisplayName}'`;
```

Comment:
Perhaps better to just omit the display name if one isn't supplied instead of setting to the resource name?

Reply:
Done [5c9f0f5](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/5c9f0f5f97f074bed7f0e7b4888779667ffaca34)